### ISSUE TITLE
Add new fields to replication

### ISSUE THREAD

**Issue by following5** [opencaching/okapi#559]
New geocache fields:
* short_descriptions
* needs_maintenance
* watchers (added today)
* oc_team_annotation (when  #558 is merged)

New log entry fields:
* images
* oc_team_entry
* needs_maintenance2
* listing_is_outdated
* location
* date_created
* last_modified

We can save ~ 250 MB of raw data at OCPL if we omit all new log fieds that have default values.

**Comment by following5** [comment]
This will insert ~ 2 GB of temporary data into the Opencaching.PL `okapi_clog` table (~1 GB at OC.DE). I guess there is enough space on the new OCPL server ...? The temp data will be deleted after 10 days.

It wont't make much a difference for OCPL if we split it up into a geocache update and a logs update, because geocaches are only ~20 % of data.

**Comment by following5** [comment]
> We can save ~ 250 MB of raw data at OCPL if we omit all new log fieds that have default values.

This also avoids about half of the `okapi_clog` entries for logs. The other half mostly have date_created > log date.

Will do the logs update first => ~ 800 MB of OC.PL clog entries; that's manageable. Then the caches (~ 400 MB) after #533 is fixed.

**Comment by following5** [comment]
Wait - it's just 500 caches with team annotations. Then we don't need to wait für #533. Let's go for 1,2 GB. :)

**Comment by following5** [comment]
We can even cut it down to 400 MB for caches only, because
* clog entries for logs are only generated when okapi_syncbase is updated, which not happens when adding new fields,
* existing clients won't expect the new fields. Who needs them, can obtain a fulldump.
